mirdata/datasets/indexes/gtzan_genre_index_1.0.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": null,
+  "version": 1.0,
```

Comment:
The index version should be the same as the dataset version right? If the dataset doesn't have an official version, should be change it to `1.0` or simply leave the index as `gtzan_genre_index.json`?

Reply:
I was hesitating about this too. I changed them all to be 1.0 for consistency, but not sure what the right answer is. I was thinking that even if they don't have an official version, it is still the "first" version. What do you think?

Reply:
Yeah, I had the same question when I introduced the version in the index for the first time... I didn't want to "make up" a version in case that was confusing, but I'm not sure if it matters to much since it's the only version this datasets have anyway

Reply:
OK, out of laziness/convenience, I'll leave them this way (null -> 1.0)